core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/OnDemandRepairSchedulerImpl.java

```diff
@@ -58,6 +60,36 @@
     private final ScheduledExecutorService myExecutor = Executors.newSingleThreadScheduledExecutor();
     private final OnDemandStatus myOnDemandStatus;
 
+    private class GetOngoingJobsThread extends Thread
+    {
+        @Override
+        public void run()
+        {
+            boolean done = false;
+            Set<OngoingJob> ongoingJobs = new HashSet<>();
+            while (!done)
+            {
+                done = true;
+                try
+                {
+                    ongoingJobs = myOnDemandStatus.getOngoingJobs(myReplicationState);
+                }
+                catch (NoHostAvailableException e)
```

Comment:
Should we write something in the log if this fails? Maybe some INFO message notifying that this is retrying and and INFO message when it is successful.